adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralRewardedAdRenderer.m

```diff
@@ -0,0 +1,141 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "GADMAdapterMintegralRewardedAdRenderer.h"
+#import "GADMAdapterMintegralUtils.h"
+#import "GADMediationAdapterMintegralConstants.h"
+#import "GADMAdapterMintegralExtras.h"
+#include <stdatomic.h>
+#import <MTGSDK/MTGSDK.h>
+#import <MTGSDKReward/MTGRewardAd.h>
+#import <MTGSDKReward/MTGBidRewardAdManager.h>
+
+@interface GADMAdapterMintegralRewardedAdRenderer ()<GADMediationRewardedAd,MTGRewardAdLoadDelegate,MTGRewardAdShowDelegate>
+
+@end
+
+@implementation GADMAdapterMintegralRewardedAdRenderer {
+    /// The completion handler to call when the ad loading succeeds or fails.
+    GADMediationRewardedLoadCompletionHandler _adLoadCompletionHandler;
+    
+    /// Data used to render an rewarded ad.
+    GADMediationRewardedAdConfiguration *_adConfiguration;
+
+    /// The Mintegral rewarded ad.
+    MTGBidRewardAdManager *_rewardedAd;
+    
+    /// An ad event delegate to invoke when ad rendering events occur.
+    id<GADMediationRewardedAdEventDelegate> _adEventDelegate;
+}
+
+- (void)renderRewardedAdForAdConfiguration:(nonnull GADMediationRewardedAdConfiguration *)adConfiguration
+                       completionHandler:(nonnull GADMediationRewardedLoadCompletionHandler)completionHandler {
+    _adConfiguration = adConfiguration;
+    // Store the ad config and completion handler for later use.
```

Comment:
Can actually remove this comment, or we should move this to above line 49. Then `_adConfiguration = adConfiguration;` should be one line below the comment.